config/initializers/keybase.rb

```diff
@@ -0,0 +1,9 @@
+module Keybase
+  def self.local_domain
+    'mastodon.social'
+  end
+
+  def self.base_url
+    'https://keybase.io'
```

Comment:
is it easy to add a run mode like `KEYBASE_DEV` which sets this to `localhost:3000` and the default mastodon port to `3001`?

Reply:
huh. good question. i'm not really sure tbh. i'll poke a little. that would allow me not to keep this branch dirty locally.